pkg/utils/constants.go

```diff
@@ -7,6 +7,39 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
 )
 
+var (
+	ImageList = []string{
+		ClusterTreeKnodeManager,
+		ClusterLinkOperator,
+		ClusterLinkAgent,
+		ClusterLinkNetworkManager,
+		ClusterLinkControllerManager,
+		ClusterLinkProxy,
+		ClusterLinkElector,
+		ClusterLinkFloater,
+		Coredns,
+		EpsProbePlugin}
+)
+
+const (
+	ClusterTreeKnodeManager      = "ghcr.io/kosmos-io/clustertree-knode-manager"
```

Comment:
I think the KnodeManager is discard

Reply:
Okay, I'll recheck the names of all the images.